sass/pages/_donate.scss

```diff
@@ -237,3 +237,21 @@ div.donate-sponsor {
     text-align: center;
     margin-top: 4rem;
 }
+
+#membership {
+  display: block;
+}
+
+#corporate-sponsorship {
+  display: none;
+}
+
+@include state-checked("donate-view") {
```

Comment:
For other reviewers, this is defined over here: https://github.com/rparrett/bevy-website/blob/76bd33f/sass/_mixins.scss#L43